build.sbt

```diff
@@ -25,9 +25,13 @@ scalacOptions := {
 )
 
 libraryDependencies := Seq(
+  "org.joda" % "joda-convert" % "1.7",
```

Comment:
I was getting compiler errors about missing stubs. I think it is a library that joda-time integrates with that for some reason is causing runtime errors when looking for converters of things. I'll try to remove this, but it seems harmless.